core/src/proof_generation/jobs.rs

```diff
@@ -226,14 +223,20 @@ pub enum ProofJob {
     /// A request has to create a new wallet
     /// The proof generation module should generate a proof of
     /// `VALID WALLET CREATE`
-    /// TODO: Remove this lint allowance
     ValidWalletCreate {
-        /// The fees to initialize the wallet with
-        fees: Vec<Fee>,
-        /// The keychain to use in the wallet
-        keys: PublicKeyChain,
```

Comment:
I know we need to circle back to fitting fees into this design, is this also true for the wallet keychain?

Reply:
I changed the job to accept the witness and statement directly, which do implicitly have this information contained in them